app/helpers/bmc_helper.rb

```diff
@@ -13,7 +13,7 @@ def power_actions
       (PowerManager::SUPPORTED_ACTIONS - ['state']).map do |action|
         display_link_if_authorized(_(action.to_s.capitalize),
                                    { :action => "power", :id => @host, :power_action => action, :auth_object => @host },
-                                   :confirm => _('Are you sure?'), :method => :put)
+                                   :data => { :confirm => _('Are you sure?') }, :method => :put)
```

Comment:
I'm not sure why Rails 4 would make us do this..? Can you point to the bootstrap docs where this is explained?